api/src/Bookmark/Model/Bookmark.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
+/*
+ * This file is part of the Spira framework.
+ *
+ * @link https://github.com/spira/spira
+ *
+ * For the full copyright and license information, please view the LICENSE file that was distributed with this source code.
+ */
+
+namespace Spira\Bookmark\Model;
```

Comment:
Move into app namespace. This is not a core model, so there is no need to have it in spira core.